R/calls_combine_by.R

```diff
@@ -12,13 +12,21 @@
 #'
 #' @examples
 #' \dontrun{
+#' fs_ch <- ChoicesFilterState$new(x = c("F", "M"), varname = "SEX")
+#' fs_ch$set_state(list(selected = c("F")))
+#'
+#' fs_rng <- RangeFilterState$new(x = 1:100, varname = "AGE")
+#' fs_rng$set_state(list(selected = c(20, 50)))
+#'
+#' fs_log <- LogicalFilterState$new(x = c(TRUE, FALSE), varname = "SURV")
+#' fs_log$set_state(list(selected = TRUE))
+#'
 #' calls <- list(
-#'   call_condition_choice("SEX", "F"),
-#'   call_condition_range("AGE", c(20, 50)),
-#'   call_condition_choice("ARM", "ARM: A"),
-#'   call_condition_logical("SURV", TRUE)
+#'   shiny::isolate(fs_ch$get_call()),
+#'   shiny::isolate(fs_rng$get_call()),
+#'   shiny::isolate(fs_log$get_call())
 #' )
-#' calls <- lapply(calls, str2lang)
+#'
```

Comment:
How about something simpler?
```
calls <- list(
  quote(SEX == "F),               # subsetting on factor
  quote(AGE >= 20 & AGE <= 50),   # subsetting on range
  quote(!SURV)                    # subsetting on logical
)
```

It's more abstract but is enough to understand this particular function.

Reply:
Would also save us the trouble of changing this example again when #187 is merged.

Reply:
Last argument is decisive